src/app/(nextjs_migration)/(authenticated)/user/breaches/page.tsx

```diff
@@ -157,6 +157,11 @@ export default async function UserBreaches() {
       <Script type="module" src="/nextjs_migration/client/js/breaches.js" />
       <Script type="module" src="/nextjs_migration/client/js/dialog.js" />
       <main data-partial="breaches">
+        <section>
+          {(process.env.PREMIUM_ENABLED === "true" && !session?.user.fxa?.subscriptions?.includes("monitor")) ?
+            <a className="button primary" href={process.env.SUBSCRIBE_PREMIUM_URL}>Subscribe to Premium</a>
+            : ''}
```

Comment:
Maybe this should be a component (`SignupButton`?), so it can be displayed or not based on some state such as whether the user is already subscribed, are not in US, and so on?

Reply:
Let's leave that for when/where we actually implement upgrades.